hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/ReplicatedMapEventPublishingService.java

```diff
@@ -106,12 +106,10 @@ public void dispatchEvent(Object event, Object listener) {
             }
 
             String mapName = ((EntryEventData) event).getMapName();
-            Boolean statisticsEnabled = statisticsMap.get(mapName);
-            if (statisticsEnabled == null) {
+            Boolean statisticsEnabled = statisticsMap.computeIfAbsent(mapName, x -> {
```

Comment:
it there any measurable impact on performance of using `computeIfAbsent` with lambda instead of explicit code?

Reply:
> it there any measurable impact on performance of using `computeIfAbsent` with lambda instead of explicit code?

I doubt it.

Anecdotally the `java.util.HashMap.computeIfAbsent(K, Function<? super K, ? extends V>)` implementation looks longer than the basic implementation here, so it's probably better.